src/signals/incident/components/IncidentForm/index.test.js

```diff
@@ -120,6 +113,19 @@ describe('<IncidentForm />', () => {
       expect(event.preventDefault).toHaveBeenCalled();
     });
 
+    it('expect to render correctly when form vars have changed', () => {
+      const incidentContainer = {
+        incident: {
+          phone: '06987654321',
+          extra_boten_geluid_meer: 'Ja! Wat een teringzooi hier',
+        },
+      };
+      wrapper.setProps({ ...props, incidentContainer });
+
+      expect(spy).toHaveBeenCalledWith(incidentContainer.incident);
```

Comment:
Can you asset that `spy` has not been called prior to changing the instance props?

Reply:
gedaan overal